src/client/public/config/openfin/launcher-local.json

```diff
@@ -1,85 +1,85 @@
 {
-  "startup_app": {
-    "name": "Reactive Ecosystem Launcher (LOCAL)",
-    "url": "http://localhost:3000/launcher",
-    "uuid": "reactive-launcher-local",
-    "defaultWidth": 650,
-    "defaultHeight": 52,
-    "defaultTop": 160,
-    "defaultLeft": 30,
-    "autoShow": true,
-    "permissions": {
-      "System": {
-        "launchExternalProcess": true
-      }
-    },
-    "saveWindowState": false,
-    "resizable": false,
-    "shadow": true,
-    "frame": false,
-    "_comment": "Openfin Excel API preloaded below + added in appAssets (not included in standard OpenFin package)",
-    "preload": [
-      {
-        "url": "http://localhost:3000/plugin/service-loader.js"
-      },
-      {
-        "url": "http://localhost:3000/plugin/fin.desktop.Excel.js"
-      }
-    ],
-    "backgroundColor": "#444C5F",
-    "alwaysOnTop": true,
-    "icon": "http://localhost:3000/static/media/icon.ico",
-    "applicationIcon": "http://localhost:3000/static/media/adaptive-mark-large.png",
-    "contextMenu": true,
-    "accelerator": {
-      "devtools": true,
-      "reload": true,
-      "reloadIgnoringCache": true,
-      "zoom": true
-    }
-  },
-  "runtime": {
-    "arguments": "--remote-debugging-port=9222",
-    "version": "13.76.44.21"
-  },
-  "shortcut": {
-    "company": "Adaptive Consulting",
-    "icon": "http://localhost:3000/static/media/icon.ico",
-    "name": "Reactive Ecosystem Launcher (LOCAL)"
-  },
-  "appAssets": [
-    {
-      "src": "http://localhost:3000/plugin/add-in.zip",
-      "alias": "excel-api-addin",
-      "version": "2.0.0",
-      "forceDownload": true
-    },
-    {
-      "src": "http://localhost:3000/plugin/LimitChecker.zip",
-      "alias": "LimitChecker",
-      "version": "1.6.0",
-      "target": "LimitChecker.application"
-    }
-  ],
-  "services": [
-    {
-      "name": "layouts",
-      "config": {
-        "features": {
-          "snap": true,
-          "dock": true,
-          "tab": false
+    "startup_app": {
```

Comment:
Can we revert this indentation change, please?

Take a look at https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig if you haven't already got it installed.